awsconfigs/apps/pipeline/s3/service-account.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ml-pipeline
+  namespace: kubeflow
+  annotations:
+    eks.amazonaws.com/role-arn: '{{ .Values.irsa.roleArn }}'
```

Comment:
this is for helmify tool purpose? if yes the templates should go into the tool folder and no {{ .Values. }} here, it should only get overrides when you run the hemify script